sources/models/src/vmware-k8s-1.20/defaults.d/50-vmware-k8s.toml

```diff
@@ -0,0 +1,71 @@
+[configuration-files.containerd-config-toml]
+# No override to path
+template-path = "/usr/share/templates/containerd-config-toml_vmware-k8s"
+
+# Kubernetes
+[services.kubernetes]
+configuration-files = [
+  "kubelet-env",
+  "kubelet-config",
+  "kubelet-kubeconfig",
+  "kubelet-bootstrap-kubeconfig",
+  "kubelet-exec-start-conf",
+  "kubernetes-ca-crt",
+  "proxy-env",
+]
+
+restart-commands = [
+  "/usr/bin/systemctl daemon-reload",
+  "/usr/bin/systemctl try-restart kubelet.service"
+]
+
+[configuration-files.kubelet-env]
+path = "/etc/kubernetes/kubelet/env"
+template-path = "/usr/share/templates/kubelet-env"
+
+[configuration-files.kubelet-config]
+path = "/etc/kubernetes/kubelet/config"
+template-path = "/usr/share/templates/kubelet-config"
+
+[configuration-files.kubelet-kubeconfig]
+path = "/etc/kubernetes/kubelet/kubeconfig"
+template-path = "/usr/share/templates/kubelet-kubeconfig"
+
+[configuration-files.kubelet-bootstrap-kubeconfig]
+path = "/etc/kubernetes/kubelet/bootstrap-kubeconfig"
+template-path = "/usr/share/templates/kubelet-bootstrap-kubeconfig"
+
+[configuration-files.kubernetes-ca-crt]
+path = "/etc/kubernetes/pki/ca.crt"
+template-path = "/usr/share/templates/kubernetes-ca-crt"
+
+[configuration-files.kubelet-exec-start-conf]
+path = "/etc/systemd/system/kubelet.service.d/exec-start.conf"
+template-path = "/usr/share/templates/kubelet-exec-start-conf"
+
+[metadata.settings.kubernetes]
+node-ip.setting-generator = "netdog node-ip"
+affected-services = ["kubernetes"]
+
+[settings.kubernetes]
+cluster-domain = "cluster.local"
+standalone-mode = false
+authentication-mode = "tls"
+pod-infra-container-image = "k8s.gcr.io/pause:3.2"
+server-tls-bootstrap = false
+cloud-provider = "external"
+
+# Metrics
+[settings.metrics]
+service-checks = ["apiserver", "chronyd", "containerd", "host-containerd", "kubelet"]
```

Comment:
A note so we don't forget to add vmtoolsd to this list after https://github.com/bottlerocket-os/bottlerocket/pull/1529